node/network/dispute-distribution/src/sender/mod.rs

```diff
@@ -122,7 +142,7 @@ impl DisputeSender {
 					ctx,
 					runtime,
 					&self.active_sessions,
-					self.tx.clone(),
+					NestingSender::new(self.tx.clone(), DisputeSenderMessage::TaskFinish),
```

Comment:
I'm missing something here. `NestingSender::new` has got `(parent: NestingSender<M, M2>, child_conversion: fn(M1) -> M2)` as input parameters. 

How `DisputeSenderMessage::TaskFinish` can be `fn(M1) -> M2`?